### PR TITLE
Chore: APP-1644 - Bump SDK Client to 0.17.1-alpha

### PR DESCRIPTION
**Pull request by Fabricevladimir**
## Description
- updates SDK to latest version
- fixes proposal caching
- fixes token formatting on DAO dashboard proposal cards (no code)

Task: [APP-1644](https://aragonassociation.atlassian.net/browse/APP-1644) & [APP-1618](https://aragonassociation.atlassian.net/browse/APP-1618)  

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the test network.

[APP-1644]: https://aragonassociation.atlassian.net/browse/APP-1644?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[APP-1618]: https://aragonassociation.atlassian.net/browse/APP-1618?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/web-app/src/utils/proposals.ts [hunk 1]
```diff
@@ -649,12 +649,19 @@ export function prefixProposalIdWithPlgnAdr(
...
-  return `${pluginAddress}_0${proposalId.slice(-5).replace(/^0+/, '') || 0}`;
+  // get proposal number "0x00" => "00"
+  parts = proposalId.split('0x');
+
+  if (parts[1] === '00') {
+    // first proposal => 0x00
+    return `${pluginAddress}_0x0`;
+  } else {
+    // other proposals => 0x3
+    return `${pluginAddress}_0x${parts[1].replace(/^0+/, '')}`;
```
Comment (WissenIstNacht) [inline packages/web-app/src/utils/proposals.ts:666]: I am uncomfortable. 😅 

This is to remove leading zeros in `parts[1]`, right? I mean if so than that's fine. But seems... fragile. Don't have a better option though. Maybe leave a comment to make sure people think about it twice before making silly changes?

**Review by WissenIstNacht — APPROVED** [review]
Thanks! One (non-critical) comment.

File: packages/web-app/src/utils/proposals.ts (hunk 1 above)
Comment (Fabricevladimir) [reply packages/web-app/src/utils/proposals.ts:666]: It freaking is fragile! And SDK randomly changes what it returns from proposal creation 😬

**Review by jamesej — APPROVED** [review]
LGTM thanks

**Pull request merged**